parrots.json

```diff
@@ -182,5 +182,29 @@
     {
       "gif": "parrotbeer.gif",
       "name": "Happy Hour Parrot"
+    },
+    {
+      "gif": "darkbeerparrot",
+      "name": "Parrot with a dark beer"
+    },
+    {
+      "gif": "blondesassyparrot",
+      "name": "Sassy Parrot with a Blonde wig"
+    },
+    {
+      "gif": "bluecluesparrot",
+      "name": "Steve from Blue's Clues Parrot"
```

Comment:
lol, we have a guy on our team that we constant joke looks like steve from blues clues, so he got his own parrot.